commands/deploy.go

```diff
@@ -283,6 +281,7 @@ Error: %s`, fprocessErr.Error())
 				TLSInsecure:             tlsInsecure,
 			}
 
+			fmt.Println(checkTLSInsecure(deploySpec.Gateway, deploySpec.TLSInsecure))
```

Comment:
My final "nit" here is `fmt.Println` will print a new-line even if the message is `""`

Shouldn't this be:

```
if msg:= checkTLSInsecure(....); len(msg) >0 {
   fmt.Println(msg)
}
```

? cc @LucasRoesler

Reply:
yes, that is right, which is why i originally proposed moving into the method.  We will need this if block everywhere to avoid the empty line

Reply:
or the option that i had initially works as well
```
fmt.Print(checkTLSInsecure(...), "\n")
```
no need for ifs just append the `\n` if needed

Reply:
Perhaps I am misunderstanding something, but

```go
fmt.Print(checkTLSInsecure(...), "\n")
```

should always print a newline. So in the case when the url starts with `https`, there will be an empty line printed.  @alexellis's comment is that we don't want an empty line printed when the TLS warning message is empty.

Reply:
Oh yes  right. I was thinking about something else.
what I can do is print the \n in the function when the warning is there and return ""
this way we can use `fmt.Print()` here and in case of no warning no new line should be printed

Reply:
I've merged the PR to bring a conclusion to the conversation, I then fixed things up in #595 along with some missing help commands.

Let's review and raise a new issue if we want further changes. 

Mario thanks for your PR. Looking forward to the next one.